<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>web_inputBookmarkNameDialog</class>
 <widget class="QDialog" name="web_inputBookmarkNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加书签</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="BodyLabel" name="BodyLabel">
       <property name="text">
        <string>书签名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="bookmarkNameLE">
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="okPb">
       <property name="text">
        <string>确定(Space)</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="cancelPb">
       <property name="text">
        <string>取消(Esc)</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
